### ISSUE TITLE
Using custom icons when using as template

### ISSUE THREAD

**Issue by Compizfox** [sardorml/vantage-typst#1]
How do I go about adding custom icons when using this as a template from Typst Universe? I.e. like

    #import "@preview/vantage-cv:1.0.0": vantage-cv, styled-link, term, skill

I've tried creating an `icons` directory in my project, and adding an icon (e.g. `phone.svg`) to it. However, when I try to use it using

    (name: "phone", link: "https://example.com")

I get the error `File not found (searched at icons/phone.svg)`. I guess that it is searching in the icons directory of the template, not of my project?

I should mention that I'm new to Typst in general, so my apologies in advance if this is a generic Typst thing I'm missing.

**Comment by sardorml** [comment]
Currently, it is not possible to use custom icons through @typst/universe package. I will have to update it to move icons directory to source folder.

**Comment by sardorml** [comment]
Yes. You are right about icons being pulled from directory of the template instead of your project. And no worries about the question.
